content/ci-cd-tool-overview.md

```diff
@@ -4,51 +4,53 @@ date: 2021-02-03
 draft: true
 ---
 
-# Intro
+## Intro 
 
-Continuous Integration (CI) and Continuous Delivery (CD) are hot topics right now with the move to the DevOps model of software development for so many organizations.
+Continuous Integration (CI) and Continuous Delivery (CD) are hot topics right now as so many organizations move to the DevOps model of software development.  <!--CK Question. Sometimes CD is Continuous Deployment, right? Do you want to acknowledge that. Like: You may see CD defined as continuous deployment...in this case, I am defining it as ... bla bla. Also I think you need to define CI/CD -- OR are the only people reading this coming from Udacity, in which case they will know what CI CD is.-->
```

Comment:
Andrew, in general I think each of the tools could use a beefier description. Also make sure to include links to all of them. I also made some edits and put some questions in like this <!-- CK question-->